### PR TITLE
fix logging vulnerabilities

### PR DESCRIPTION
**Pull request by michal-cmarada**
Signed-off-by: Michal Cmarada <michal.cmarada@pantheon.tech>

### CODE REVIEW

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ private void loadPolicyClassPath(Model model, Helper.loadPolicyLineHandler<Strin
             is.close();
             br.close();
         } catch (IOException e) {
-            e.printStackTrace();
+            LOG.error("IO error occurred");
```
Comment (rovarga) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java]: Also log exception cause...

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ private void loadPolicyClassPath(Model model, Helper.loadPolicyLineHandler<Strin
...
             throw new Error("IO error occurred");
```
Comment (rovarga) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java]: While we are here, change this to IllegalStateException and chain the cause

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java (hunk 1 above)
Comment (michal-cmarada) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java]: fixed in next commit

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java (hunk 2 above)
Comment (michal-cmarada) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java]: fixed in next commit

**Pull request merged**